proto/dydxprotocol/vault/tx.proto

```diff
@@ -39,6 +39,23 @@ message MsgDepositToVault {
 // MsgDepositToVaultResponse is the Msg/DepositToVault response type.
 message MsgDepositToVaultResponse {}
 
+// MsgWithdrawFromVault withdraws the specified amount of shares from the vault to the subaccount.
+message MsgWithdrawFromVault  {
+  option (cosmos.msg.v1.signer) = "subaccount_id";
+  
+  // The vault to withdraw from.
+  VaultId vault_id = 1;
+  
+  // The subaccount to withdraw to.
+  dydxprotocol.subaccounts.SubaccountId subaccount_id = 2;
+  
+  // Number of shares to withdraw.
+  uint64 shares = 3;
```

Comment:
we use type [NumShares](https://github.com/dydxprotocol/v4-chain/blob/main/proto/dydxprotocol/vault/vault.proto#L27) to represent shares

Reply:
We defined `NumShares` to 
1. use serializable int (as shares can get quite large)
2. define a message type to be used in cosmos store

Good catch in that link, it should be NumShares. I'll fix it in another PR